src/visual-regression-app/src/components/test-case/image-diff/fullscreen-diff/fullscreen-diff.ts

```diff
@@ -11,6 +11,8 @@ import '@sbb-esta/lyne-elements/chip-label.js';
 import '@sbb-esta/lyne-elements/radio-button.js';
 import type { SbbRadioButtonGroupElement } from '@sbb-esta/lyne-elements/radio-button/radio-button-group/radio-button-group.js';
 
+type DiffFileType = 'baselineFile' | 'failedFile' | 'diffFile';
```

Comment:
We should probably export types we publicly use
```suggestion
export type DiffFileType = 'baselineFile' | 'failedFile' | 'diffFile';
```